src/chunk.js

```diff
@@ -270,28 +267,16 @@ class StartupTask extends Task {
 
   /** @override */
   useRequestIdleCallback_() {
-    // We only start using requestIdleCallback when the viewer has
-    // been initialized. Otherwise we risk starving ourselves
-    // before we get into a state where the viewer can tell us
-    // that we are visible.
-    return !!this.chunks_.viewer;
+    // DO NOT SUBMIT: is this ok?
+    return true;
```

Comment:
No, it probably isn't. The reason is that we need to go fast until the viewer can tell us we are visible. Otherwise we fail to install the viewer and may need to get a long time to get there. I think that is still the case.

Reply:
We no longer need viewer to know that a document is visible/invisible.  Does this reasoning still apply in this case?

Reply:
Fixed.